app/services/audit_results.rb

```diff
@@ -154,9 +154,9 @@ def report_results(logger=Rails.logger)
       log_result(r, logger)
       if r.key?(INVALID_MOAB)
         msg = "#{workflows_msg_prefix} || #{r.values.first}"
-        WorkflowReporter.report_error(druid, 'moab-valid', msg)
+        WorkflowReporter.report_error(druid, actual_version, 'moab-valid', msg)
```

Comment:
providing a version (when avail) means workflow service doesn't need to look it up)